mlaut/benchmarking/tests/test_orchestrator.py

```diff
@@ -0,0 +1,83 @@
+import os
+import pandas as pd
+import numpy as np
+from sklearn.metrics import accuracy_score
+from mlaut.benchmarking.data import RAMDataset
+from sklearn import datasets
+from mlaut.highlevel.tasks import CSCTask, CSRTask
+
+from mlaut.benchmarking.results import HDDResults, RAMResults
+from mlaut.highlevel.strategies import CSCStrategy
+from mlaut.model_selection import SingleSplit
+from mlaut.benchmarking.orchestration import Orchestrator
+
+import pytest
+
+from sklearn import linear_model
+from mlaut.strategies.baseline import BaselineClassifierStrategy, BaselineRegressorStrategy
+from mlaut.strategies.bayes import GaussianNaiveBayesStrategy, BernoulliNaiveBayesStrategy
+from mlaut.strategies.cluster import KNeighboursStrategy
+from mlaut.strategies.decision_trees import DecisionTreeClassifierStrategy, DecisionTreeRegressorStrategy
+from mlaut.strategies.ensemble import (RandomForestClassifierStrategy,
+                                       RandomForestRegressorStrategy,
+                                       GradientBoostingClassifierStrategy,
+                                       GradientBoostingRegressorStrategy, 
+                                       BaggingClassifierStrategy, 
+                                       BaggingRegressorStrategy)
+from mlaut.strategies.glm import (LinearRegressonStrategy, 
+                                  RidgeRegressionStrategy, 
+                                  LassoStrategy, 
+                                  LassoLarsStrategy, 
+                                  LogisticRegressionStrategy, 
+                                  BayesianRidgeStrategy, 
+                                  PassiveAggressiveClassifierStrategy)
+
+from mlaut.strategies.svm import SVMStrategy, SVRStrategy
+from mlaut.strategies.neural_networks import KerasClassificationStrategy, KerasRegressionStrategy
+
+from mlaut.benchmarking.evaluation import Evaluator
+from mlaut.benchmarking.metrics import PairwiseMetric
+from sklearn.metrics import accuracy_score
+
+iris = datasets.load_iris()
+wine = datasets.load_wine()
+def test_orchestrator():
```

Comment:
this test checks if the orchestrator runs with no errors, but doesn't test if results are as expected for the given datasets/estimators, I think that's very important and we should have at least a test that does that for a single method, ie comparing our "automated" orchestrator against manual orchestration

Reply:
will add one